integration-tests/bootstrap/bootstrap_suite_test.go

```diff
@@ -0,0 +1,50 @@
+package bootstrap
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	kubeConfigPath *string
+	kubeConfig     *kubernetes.Clientset
+)
+
+func homeDir() (string, error) {
```

Comment:
move this outside this package so that this can be reused.

Reply:
added a new package called kubeconnect which help in fetching the config and placed it there.